domains/drippypop.json

```diff
@@ -0,0 +1,14 @@
+{
+    "owner": {
+        "username": "TheBlockbreaker",
+        "email": "burstsgodly@gmail.com"
+    },
+    "record": {
+        "A": [
+            "217.174.245.249",
+            "51.161.54.161"
+            ],
+        "MX": ["mail.is-a.dev"],
+        "TXT": "v=spf1 mx a:mail.is-a.dev ~all"
+    }
```

Comment:
on a side note, we dont support is-a.dev hosting anymore

Reply:
what i mean is is-a.dev shutdown, so you cant make domains with these records anymore